main/daml/Rental.daml

```diff
@@ -84,11 +87,19 @@ template Property
           tenant: Party 
           start: Time 
           end: Time 
-          rate: Decimal 
```

Comment:
I think this breaks the `OfferToTentant` exercise that exists in the [`IndicationOfInterest` `MakeOffer` choice](https://github.com/LeventeBarczyDA/daml-bnb/pull/3/files#diff-e11cba370a31915db8c6d4ddbf52d6814ac11ba15636fe72ca5065a8a3c51a53R145-R150). That takes a rate that's now unused.